http/headers/Link.json

```diff
@@ -34,6 +34,119 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "blocking": {
+          "__compat": {
+            "spec_url": "https://html.spec.whatwg.org/multipage/semantics.html#attr-link-blocking",
+            "tags": [
+              "web-features:blocking-render"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "105"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false,
+                "impl_url": "https://bugzil.la/1751383"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false,
+                "impl_url": "https://webkit.org/b/267232"
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "expect": {
+          "__compat": {
+            "description": "rel=expect",
+            "spec_url": "https://html.spec.whatwg.org/multipage/links.html#link-type-expect",
+            "support": {
+              "chrome": {
+                "version_added": "124",
+                "impl_url": "https://crbug.com/1507845"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "fetchpriority": {
```

Comment:
Mirrors the `<link>` element for these features, which were added after introduction of LINK header. Ditto other subfeatures above. Assumption is anything that was implemented in link element is also in the header as per the docs. This is definitely true for fetchpriority on FF.

Reply:
I think there are _potentially_ differences- for a start, the Link header appears much more recently than the tag (thought that is captured), and more generally, there are things supported on the tag that are not supported via this header.

IMO it matches "policy" to document these as I have done.

Josh points out in discussion the bcd channel that just documenting the ones that come in different versions is confusing for users - i.e. his take is that once you have added one subfeature you should add all of them. I agree, but that also is against the policy.

Upshot, I don't plan to change this unless @Elchi3 or @queengooborg can confirm and update policy to reflect that.